package_viviware_index.json

```
@@ -2,10 +2,9 @@
   "packages": [
     {
       "name": "viviware",
-      "maintainer": "VIVITA, Inc.",
+      "maintainer": "VIVIWARE JAPAN, Inc.",
+      "email": "info@viviware.com",
```

@dodo5522 Slackの以下のスレで、メールは `viviware-support@viviware.com` とすることになりそうです。
https://vivita.slack.com/archives/C0ZRW2UMV/p1624346105206700?thread_ts=1621583798.065600&cid=C0ZRW2UMV

```suggestion
      "email": "viviware-support@viviware.com",
```